andela/ah-frontend-poseidon
# #162162956 setup redux

**What does this PR do?**
- Setup React/Redux

**Description of the tasks to be completed?**
- Create a file structure for Redux workflow.
- Create the Redux store

**How should this be manually tested?**
- Clone the repository by running `git clone https://github.com/andela/ah-frontend-poseidon.git`
- Run `git checkout ch-setup-redux-162162956` and you should see the `redux` folder under the `src` folder

**What are the relevant pivotal tracker stories?**
#162162956
